Modules/_ssl.c

```diff
@@ -5183,6 +5246,12 @@ PyInit__ssl(void)
     if (PySSLErrorObject == NULL)
         return NULL;
 
+    /* ssl.CertificateError used to be a subclass of ValueError */
+    bases = PyTuple_Pack(2, PySSLErrorObject, PyExc_ValueError);
```

Comment:
`PyTuple_Pack` does not incref elements, but `Py_DECREF(bases)` will. Should probably avoid that here (`Py_BuildValue("OO", ...)` will incref)

Reply:
thx